css/index.css

```diff
@@ -111,7 +140,36 @@ p {
     align-items: center;
     justify-content: center;
 }
-
+.middle-content .boxes .box:nth-child(1){
```

Comment:
This worked! Another method would be to assign the same class to all of the boxes and a different ID to each box in HTML.